test-cases/terraform/aws/iam/iam-entities/role_assume_policy_principal_all/main.tf

```diff
@@ -0,0 +1,47 @@
+# Two roles, both are bad and should be caught
```

Comment:
So I assume (no pun intended) you qualify this test as "passed" only if the tool catches both roles? In this case it might be worth clarifying as a comment

Reply:
@christophetd added a sentence there, can you take a look?